### PR TITLE
[ADD] module mrp_calendar_view

### PR DESCRIPTION
**Pull request by hurrinico**
As @pedrobaeza says in https://github.com/OCA/rma/pull/18 i've moved my module here.

### CODE REVIEW

File: mrp_calendar_view/view/mrp_repair_view.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <field name="arch" type="xml">
+            <calendar string="Repair Orders Calendar" date_start="date_repair" color="user_id">
+                <field name="product_id"/>
+                <field name="partner_id"/>
+            </calendar>
+        </field>
+    </record>
+
+    <record id="mrp_repair.action_repair_order_tree" model="ir.actions.act_window">
+        <field name="name">Repair Orders Calendar</field>
```
Comment (pedrobaeza) [inline mrp_calendar_view/view/mrp_repair_view.xml]: Don't add these lines. Only 'view_mode' one is needed.

Comment (hurrinico) [inline mrp_calendar_view/view/mrp_repair_view.xml]: Changed, Thanks

**Comment by pedrobaeza** [comment]
Thanks!

:+1:

**Comment by archetipo** [comment]
:+1:

**Comment by oihane** [comment]
:+1:

**Pull request merged**